api/handlers_ltm.go

```diff
@@ -162,3 +167,35 @@ func (s *server) CreateClientSSLProfile(w http.ResponseWriter, r *http.Request)
 	w.WriteHeader(http.StatusOK)
 	w.Write(out)
 }
+
+// DeleteClientSSLProfile deletes SSL Client Profile and cert-key pair
+func (s *server) DeleteClientSSLProfile(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	host := vars["host"]
+	name := vars["name"]
+
+	log.Infof("delete client-ssl profile %s on host %s", name, host)
+
+	ltmService, ok := s.LTMServices[host]
+	if !ok {
+		msg := fmt.Sprintf("LTM host service not found for account: %s", host)
+		handleError(w, apierror.New(apierror.ErrNotFound, msg, nil))
+		return
+	}
+
+	orch := &ltmOrchestrator{
+		client: ltmService,
+	}
+
+	if err := orch.deleteClientSSLProfile(r.Context(), name); err != nil {
+		handleError(w, err)
+		return
+	}
+
+	out := []byte(fmt.Sprintf("deleted client-ssl profile %s on host %s", name, host))
```

Comment:
minor, but I would just put this inside of the `w.Write()` instead of creating the `out` variable.